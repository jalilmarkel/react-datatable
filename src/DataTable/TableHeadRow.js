import styled from 'styled-components';

const TableHeadRow = styled.div`
  display: flex;
  width: 100%;
`;

export default TableHeadRow;
