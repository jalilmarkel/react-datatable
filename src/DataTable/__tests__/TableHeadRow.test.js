import 'jest-styled-components';
import React from 'react';
import { render, cleanup } from 'react-testing-library';
import TableHeadRow from '../TableHeadRow';

afterEach(cleanup);

test('component <TableHeadRow /> should render correctly', () => {
  const { container } = render(<TableHeadRow />);
  expect(container.firstChild).toMatchSnapshot();
});
