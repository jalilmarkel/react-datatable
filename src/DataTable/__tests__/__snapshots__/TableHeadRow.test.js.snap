// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component <TableHeadRow /> should render correctly 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  width: 100%;
}

<div
  className="c0"
/>
`;
